infra/aws/terraform/prow-build-cluster/resources/kubecost/prod-cluster-values.yaml

```diff
@@ -1,6 +1,17 @@
 ## save this to values.yaml file and run
 ## k create configmap kubecost-helm-values --from-file=values.yaml -n kubecost
 ## on the prow-build-cluster
+global:
+  prometheus:
+    enabled: true
+  grafana: # prometheus metrics will be local cluster only, disable grafana to save resources
+    enabled: false
+    proxy: false
+tolerations:
```

Comment:
Does KubeCost support setting nodeSelector? Otherwise, KubeCost pods _might_ get scheduled to the stable nodes, but it's not guaranteed.

Reply:
This is only relevant for KubeCost components that are not DaemonSets (e.g. `kubecost-network-costs` should run on all nodes)

Reply:
Good point, sure it does.